docs/custom-domain/registry-setup-deprecated.md

```diff
@@ -0,0 +1,56 @@
+# How to use a custom domain for azure container registry
+
```

Comment:
Is the content in this article still needed?

Reply:
@sajayantony ?

Reply:
Yes. that should be removed since we now support custom domains through a support ticket. I would rather we remove the nginx doc delete the template/commands.

Reply:
Okay, removed